src/OcticonsPromo.js

```diff
@@ -26,7 +27,7 @@ export default function OcticonsPromo() {
         </Box>
       </Box>
 
-      <Box className="container-xl mx-auto" color="blue.2">
+      <Box mx="auto" className="container-xl" color="blue.2">
```

Comment:
This is probably my bad for adding the class, but `container-xl` applies `margin-right: auto; margin-left: auto;`. I'm not sure if we'll want to keep that when we add container props to primer/components though so cool if you leave this in.